lib/chat_app_web/live/home_live.ex

```diff
@@ -37,4 +42,35 @@ defmodule ChatAppWeb.HomeLive do
     </div>
     """
   end
+
+  def handle_event(
+        "create_room",
+        %{
+          "room_name" => room_name,
+          "owner_name" => owner_name,
+          "is_private" => is_private,
+          "password" => password
+        },
+        socket
+      ) do
+    is_private =
+      if is_private == "on" do
+        true
+      else
+        false
+      end
```

Comment:
I know this is not incredibly important, however I think this if could be replaced with a simple case statement, something like this:
```elixir
is_private = 
  case is_private do 
    "on" -> true
    _ -> false
  end
```

But really, I also used an if statement if I'm not mistaken, so it's not essential to remove it.

Reply:
I can try with that as well, the syntax is weird enough as it is